src/components/form/button.ts

```diff
@@ -10,7 +10,7 @@ type SharedButtonStyleProps = {
     width?: string
     primary?: string
     secondary?: string
-    tertiary?: string
+    tertiary?: string | boolean
```

Comment:
This is already fixed in another PR. Let's remove this change :pray:

Reply:
but throws error on mine

Reply:
Because that [PR](https://github.com/binary-com/deriv-com/pull/3354) is not merged yet. Anyway it will be merged soon. By removing this change we can avoid conflicts.

Reply:
Is it thesame fix with this PR or different?

Reply:
it was throwing error in my pr during typescript refactoring.so i added the type for button.. @mohammad-rework has done it in different pr.i can remove mine